src/status_im/contexts/wallet/add_account/create_account/events.cljs

```diff
@@ -80,6 +100,21 @@
 
 (rf/reg-event-fx :wallet/get-derived-addresses get-derived-addresses)
 
+(rf/reg-event-fx
+ :wallet/next-derivation-path
+ (fn [_ [{:keys [on-success keypair-uid]}]]
+   {:fx [[:json-rpc/call
+          [{:method     "accounts_resolveSuggestedPathForKeypair"
+            :params     [keypair-uid]
+            :on-success on-success
+            :on-error   (fn [error]
+                          (log/error
+                           "Failed to resolve next path derivation path"
+                           {:event  :wallet/next-derivation-path
+                            :method "accounts_resolveSuggestedPathForKeypair"
+                            :error  error
+                            :params keypair-uid}))}]]]}))
```

Comment:
This is a new call, we need it to properly avoid 
- #19531

Reply:
@ulisesmac was this the root of the problem?

Reply:
The root of the problem to fix the derivation path, yes.

But it's not the root of the problem to properly create new keypairs.